rmf_traffic/src/rmf_traffic/geometry/SimplePolygon.cpp

```diff
@@ -655,10 +655,16 @@ void SimplePolygon::insert_point(
 //==============================================================================
 FinalShape SimplePolygon::finalize() const
 {
-  //TODO calculate the characteristic length for SimplePolygon
+  double characteristic_length = 0;
+  for (const Eigen::Vector2d& point : this->get_points())
+  {
+    double distance = (point - Eigen::Vector2d{0, 0}).norm();
```

Comment:
This is equivalent to `const double distance = point.norm();`

Reply:
Noted.